gallery/visualtests/nxButton.js

```diff
@@ -114,8 +115,20 @@ describe('NxButton', function() {
           focusAndHoverTest(selector));
     };
 
+    describe('Small NxButton section', function() {
+      const selector = '#nx-button-small-example .nx-btn--small:first-child';
+
+      it('has a white background by default', simpleTest(selector));
+      it('has a light blue background when hovered', hoverTest(selector));
+      it('has a light blue background when clicked', clickTest(selector));
+      it('has a blue inner outline when focused', focusTest(selector));
```

Comment:
Looks more like a grey inner outline when focussed for the light theme
![image](https://github.com/user-attachments/assets/b8771f7e-39f9-482c-b72a-6f6bfa66eee0)

but does look light blue for the dark theme
![image](https://github.com/user-attachments/assets/e1ac834e-7b49-41d6-87ae-9e2599ee8197)

Reply:
it gives me a blueish feeling. but yes it is grey when i use color picker ;)